ga4gh/datamodel/variants.py

```diff
@@ -477,6 +477,22 @@ def getNumVariants(self):
         # TODO How do we get the number of records in a VariantFile?
         return 0
 
+    def _updateCallSetIds(self, variantFile):
+        """
+        Updates the call set IDs based on the specified variant file.
+        """
+        checkConsistency = True
+        if self._callSetIds is None:
+            self._callSetIds = []
+            checkConsistency = False
+
+        for sample in variantFile.header.samples:
+            if checkConsistency:
+                if sample not in self._callSetIds:
+                    raise Exception("Inconsistent sample names in VCF")
+            else:
+                self._callSetIds.append(sample)
```

Comment:
Can we add this check please @shajoezhu? I think we should be ready to merge then, and add the extra issues for us to follow up on.